.ci/test.sh

```diff
@@ -116,32 +121,31 @@ if [[ $TASK == "lint" ]]; then
 fi
 
 if [[ $TASK == "check-docs" ]] || [[ $TASK == "check-links" ]]; then
-    cd "${BUILD_DIRECTORY}/docs"
     conda env create \
-        -n $CONDA_ENV \
-        --file ./env.yml || exit 1
+        -n "${CONDA_ENV}" \
+        --file ./docs/env.yml || exit 1
     conda install \
         -q \
         -y \
         -n $CONDA_ENV \
             'doxygen>=1.10.0' \
             'rstcheck>=6.2.4' || exit 1
+    # shellcheck disable=SC1091
     source activate $CONDA_ENV
     # check reStructuredText formatting
-    cd "${BUILD_DIRECTORY}/python-package"
-    rstcheck --report-level warning $(find . -type f -name "*.rst") || exit 1
-    cd "${BUILD_DIRECTORY}/docs"
-    rstcheck --report-level warning --ignore-directives=autoclass,autofunction,autosummary,doxygenfile $(find . -type f -name "*.rst") || exit 1
+    find "${BUILD_DIRECTORY}/python-package" -type f -name "*.rst" \
+        -exec rstcheck {} \+
```

Comment:
Seems that `--report-level warning` and `|| exit 1` parts were lost.

Reply:
Indeed, good spots (with the one just below!)